### ISSUE TITLE
Resolve existing "orphaned resource" issues & determine if further work needed

### ISSUE THREAD

**Issue by paynejd** [OpenConceptLab/DATIM-Metadata-Project#180]
An "Orphaned Resource" is a concept/mapping/reference that exists but that is not part of a repo version (and therefore not part of an export).

We know that this happened in the past sometimes when the processing of a repo version stopped prematurely for some reason. However, when a new repo version is created, if resources are copied from the previous repo version, then the "orphaning" of a resource would be propagated to subsequent repo versions unless manually resolved.

The goal is to identify and resolve all existing orphaned resources and then monitor the situation to see if it happens again. If it continues to happen, we should determine what is needed to prevent it from reoccurring.

**Comment by paynejd** [comment]
@rkorytkowski Adding you to this so that we can discuss. I'm assuming that we'll need some server work that you do and sleuthing around in the UI/API that I can do. See also #177

**Comment by rkorytkowski** [comment]
What do we do with orphaned resources? Do we attach them all to the head of source?

**Comment by paynejd** [comment]
Yes, reattaching to the head of source (and maybe collection, if relevant?)
is the right solution.
